services/rfq/api/versions.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I would remove this file just for DRY purposes

Reply:
did not mean to leave that here - thanks!